src/test/scala/com/github/dnvriend/serializer/avro4s/AvroSENoOldClassesTest.scala

```diff
@@ -68,172 +68,47 @@ class AvroSENoOldClassesTest extends AvroTestSpec {
   }
 
   "v1 to v2" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV2]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wf"),
-      oldSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 1,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" }
-        |  ]
-        |}
-      """.stripMargin.toSchema,
-      newSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 2,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" },
-        |   { "name" : "director", "type" : "string", "default" : "unknown" }
```

Comment:
The JSON schema I used in my first commit are taken directly from avro4s output and did not included the default value for the `director` field. I will check why avro4s does not create defaults. This is an inconsistency to be aware.